tests/modules/sitemap/test-sitemap.php

```diff
@@ -57,6 +57,37 @@ public function test_only_pages() {
 		);
 	}
 
+	/**
+	 * Creates pages, marks some as NOINDEX and tests whether only the non-NOINDEX pages are being shown in the sitemap.
```

Comment:
Do you think my format might help you with documenting unit tests? Especially for future reference.

https://github.com/semperfiwebdesign/all-in-one-seo-pack/blob/3.0/tests/base/class-aioseop-notices-testcase.php#L319

Reply:
good idea. have used that.

Reply:
Looks good, but isn't `Actual: As expected; no current issue.` suppose to mention the issue it is fixing? It helps to show why the test was initially made. I thought this was more of a bug where there was an unintended result occuring.

The other PR I linked was new code, so there was no prior actual for me to add.